doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -261,9 +263,9 @@ and attributes, that can be used in the operations that support the `canonical`
 
 In the case of being present, the error payload is JSON object including the following fields:
 
-* `error` (mandatory): a textual description of the error.
-* `description` (optional): additional information about the error.
-* `affectedItems` (optional): a list of elements affected by the error. Depending on the operation, it may
++ `error` (mandatory): a textual description of the error.
++ `description` (optional): additional information about the error.
++ `affectedItems` (optional): a list of elements affected by the error. Depending on the operation, it may
   refer to entities, registrations or subscriptions.
```

Comment:
Hi Ken,

Good point. Attributes can only be a subsection at certain parts of the document (Request, Response ... Resource). However API Blueprint allows a Data Structures section which unfortunately is not currently rendered by the Apiary implementation. 

Nonetheless, I have aligned the description as much as possible with the Attributes format, just to make it clearer